pkg/strace/strace_test.go

```diff
@@ -39,4 +39,7 @@ func TestBasicStrace(t *testing.T) {
 	}
 
 	go c.RunTracerFromCmd(exec.Command("ls", "/etc/hosts"))
+	for range c.Records {
```

Comment:
not sure what this is for

Reply:
RunTracerFromCmd sends stuff on this channel. It'll close it when all processes exit. This is to make sure that RunTracerFromCmd actually finishes & exits before we exit the test process.